pyscriptjs/tests/test_01_basic.py

```diff
@@ -21,7 +22,8 @@ def test_pyscript_hello(self):
         """
         self.write("hello.html", doc)
         self.goto("hello.html")
+        # I wonder if we need to call wait_for_load in every single test?
+        # It's only relevant to call it on tests that are interfering somehow with the way we load the page, IMO
         wait_for_load(self.page)
```

Comment:
I guess that for the vast majority of tests the answer is "yes". Usually, we will want to do the following:
1. write an HTML page with one or more `<py-script>` (and other `<py-*>`) tags
2. load it in the browser and wait for all the tags to be executed
3. look at the results and check that they did what we expect

And we cannot do (3) before the page is fully loaded.
But this does not mean that we have to call `wait_for_load` manually in all tests. Probably we could put it inside `self.goto`, but not now because else we break `test_00_support.py` (which doesn't load pyscript at all, and thus if you call `wait_for_load` it would wait forever).

There might be some very specific corner cases in which we might want to see what happens to the page *before* it has finished to load: for example if we want to check the behavior of the splash screen. But as said, this is a corner case.